src/databricks/labs/ucx/assessment/aws.py

```diff
@@ -455,3 +533,68 @@ def _run_command(self, command: str):
     @staticmethod
     def _get_json_for_cli(input_json: dict) -> str:
         return json.dumps(input_json).replace('\n', '').replace(" ", "")
+
+
+@dataclass()
+class AWSGlue:
+    aws_region: str
+    aws_account_id: str
+
+    @classmethod
+    def get_glue_connection_info(
```

Comment:
Typically, we introduce a `Maybe<object>` for this where we parse something to create the object. And we yield the object when successfully parsed, we yield the failure when not successfully parsed, see `MaybeGraph` and `MaybeTree` as example